fr.henri2h.minestrix.json

```diff
@@ -0,0 +1,137 @@
+{
+    "app-id": "fr.henri2h.minestrix",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "43",
+    "sdk": "org.gnome.Sdk",
+    "command": "minestrix",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--device=all",
+        "--filesystem=xdg-download",
+        "--talk-name=org.gnome.Notifications"
+    ],
+    "build-options": {
+        "build-args": [
+            "--share=network"
```

Comment:
build args are not allowed. You need to make sure the build is offline.

Reply:
done